CONTRIBUTING.md

```diff
@@ -0,0 +1,59 @@
+# Contributing
+
+## Translations
+
+We welcome contributions for translating our application into various languages. If you'd like to help with translations, please follow these steps.
+
+### Steps
+
+1. Look for the `/utils/translations` directory to find existing translations. If you don't find the language you wish to translate the application into, proceed to step 2.
+
+2. Create a new file for the language by duplicating the `ru.ts` file in the `/translations` directory. Name it using the language code (e.g., `fr.ts` for French translations).
+
+3. Translate the content within the file into your desired language.
+
+4. In the directory `/utils` open the file `translation.ts`. Import the generated file like this:

```

Comment:
Import translation file in  the `./src/utils/translations.ts`

Reply:
Fixed